docs/advanced-topics/custom-serialization.md

```diff
@@ -30,7 +32,7 @@ public static class SerializationExtensions
 
     public static void WriteValueSafe(this FastBufferWriter writer, in Url value)
     {
-        writer.WriteValueSafe(instance.Value);
+        writer.WriteValueSafe(value.Value);
```

Comment:
minor nit-pick.
could we change this:
`in Url value`
To something like this:
`in Url urlValue`

(just trying to avoid `value.Value`)